content/v2.0/cloud/account-management/billing.md

```diff
@@ -14,7 +14,9 @@ menu:
 
     {{< nav-icon "cloud" >}}
 
-    Complete the following procedures as needed:
+  If you subscribed to a plan through AWS Marketplace, click the **AWS** link to access your billing and subscription information.
```

Comment:
should we add the Manage Marketplace Subscriptions link in docs? (feels maybe a bit much since it's in UI) https://aws.amazon.com/marketplace/library/

Reply:
I think this note is good enough, but you may want to bold "AWS Marketplace" so it's easy to see when scanning quickly.

Reply:
The line below this change is now a bit misleading - it says "If you subscribed to an Influxdata plan ..." Even if they went through the AWS marketplace, they are still subscribing to an Influxdata plan. Maybe we can make it clearer that it's an either/or - either you went through AWS or you subscribed directly with Influx.